### PR TITLE
Create isolated-vm toggle

### PR DESCRIPTION
**Pull request by adrinr**
## Description
Add a toggle to enable isolated-vm. By default this will be off, allowing us to run a controlled isolated-vm release

## Feature branch env
[Feature Branch Link](http://fb-reenable-isolatedvm.fb.qa.budibase.net)

### CODE REVIEW

File: packages/server/src/environment.ts [hunk 1]
```diff
@@ -126,20 +127,26 @@ const environment = {
   getDefaults: () => {
     return DEFAULTS
   },
+  ISOLATEDVM_QUERY_TRANSFORMERS: !!process.env.ISOLATEDVM_QUERY_TRANSFORMERS,
```
Comment (mike12345567) [inline packages/server/src/environment.ts:130]: can we keep the environment variables together above the final exports/functions in the `environment.ts` files? Makes it easier to not miss some environment variables in the list!

**Review by mike12345567 — APPROVED** [review]
LGTM

File: packages/server/src/jsRunner/tests/jsRunner.spec.ts [hunk 2]
```diff
@@ -24,18 +9,28 @@ const { runJsHelpersTests } = require("@budibase/string-templates/test/utils")
...
 
-describe("jsRunner", () => {
+describe.each([
+  ["vm", false],
+  ["isolated-vm", true],
+])("jsRunner (using %s)", (_, useIsolatedVM) => {
   const config = new TestConfiguration()
 
   beforeAll(async () => {
+    environment._set("ISOLATEDVM_JS_RUNNER", useIsolatedVM)
```
Comment (samwho) [inline packages/server/src/jsRunner/tests/jsRunner.spec.ts:23]: This is going to leak out of these tests and into other tests. I recommend using `config.withEnv`, which makes sure to set values back to what they were prior to being overridden.

Comment (mike12345567) [reply packages/server/src/jsRunner/tests/jsRunner.spec.ts]: Removed this.

**Pull request merged**